src/WebUI/WorkLoads/WorkloadDetails.scss

```diff
@@ -0,0 +1,35 @@
+@import "../Common/Common.scss";
+
+.workload-details-column-header {
+    font-size: 14px;
+}
+
+.workload-details-primary-text {
+    font-size: 16px;
+    font-weight: $fontWeightSemiBold;
+    padding-left: 20px;
+}
+
+.workload-details-secondary-text {
+    padding-left: 20px;
+    color: $secondary-text;
+    font-weight: $fontWeightSemiBold;
+}
+
+.w-key {
```

Comment:
too genric and not part of any parent key

Reply:
Changed the classnames to  bemore specific